.github/workflows/pull-request.yaml

```diff
@@ -12,6 +12,9 @@ jobs:
       CI_TOOLS_URL: https://github.com/samuelattwood/partner-charts-ci/releases/latest/download/partner-charts-ci-linux-amd64
 
     steps:
+      - uses: actions/setup-node@v2
+        with:
+           node-version: '16.x'
       - uses: actions/checkout@v2
```

Comment:
I think the fix is to update this action to [the latest version](https://github.com/marketplace/actions/checkout).

Reply:
I'm almost sure this fix the warning message. More references https://github.com/actions/checkout/issues/1047#issuecomment-1355170319

Reply:
`actions/checkout@v2` [uses Node12](https://github.com/actions/checkout/blob/v2.0.0/action.yml#L33).

Reply:
This could be a possible fix, but I'm not familiar with the reason behind the use of `actions/checkout@v2` in this pipeline, so just to be on the save side I have opted out for this solution.

Reply:
@atrendafilov This is the fix according to the documentation and references. In fact, we don't need to add a new step.